packaging/conda/bld.bat

```diff
@@ -3,13 +3,20 @@ REM Echo all output
 
 REM Build the Khiops binaries
 cmake --preset windows-msvc-release -DBUILD_JARS=OFF -DTESTING=OFF
-cmake --build --preset windows-msvc-release --parallel --target MODL MODL_Coclustering KhiopsNativeInterface KNITransfer 
+cmake --build --preset windows-msvc-release --parallel --target MODL MODL_Coclustering KhiopsNativeInterface KNITransfer _getprocnumber
 
 REM Copy the MODL binaries to the Conda PREFIX path
 mkdir %PREFIX%\bin
 copy build\windows-msvc-release\bin\MODL.exe %PREFIX%\bin
 copy build\windows-msvc-release\bin\MODL_Coclustering.exe %PREFIX%\bin
+copy build\windows-msvc-release\bin\_getprocnumber.exe %PREFIX%\bin
 copy build\windows-msvc-release\bin\KNITransfer.exe %PREFIX%\bin
-copy build\windows-msvc-release\bin\KhiopsNativeInterface.dll %PREFIX%\lib
+copy build\windows-msvc-release\bin\KhiopsNativeInterface.dll %PREFIX%\bin
+copy build\windows-msvc-release\lib\KhiopsNativeInterface.lib %PREFIX%\lib
```

Comment:
Could explain these changes?
- copy the DLL into `bin`
- add a static library to `lib`?

Reply:
In windows, DLL are associated with a .lib file see documentation [here](https://learn.microsoft.com/en-us/troubleshoot/windows-client/setup-upgrade-and-drivers/dynamic-link-library#load-time-dynamic-linking). In the kni package, the .dll is in bin and the .lib is in the bin. I have simply followed this...

Reply:
comment is updated

Reply:
Ok, so from what I understood, the static is need at load-time only to enable the DLL loading to get the addresses of the functions exported by the DLL.

Reply:
However, I did not understand the "the .lib is in the bin" part: I see it copied to `lib` in the build recipe...